library/src/main/res/layout/license.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:tools="http://schemas.android.com/tools"
+             android:layout_width="match_parent"
+             android:layout_height="wrap_content"
+  >
+
+  <!--
+  Wrap this LL in a FL to circumvent gone bug where a top-level LL will still be used in onMeasure
+  event though it's marked as gone.
```

Comment:
Is there any link to the issue, like stackoverflow or google issue tracker?
including those links would be helpful.

Reply:
Done.